.gitignore

```diff
@@ -0,0 +1,109 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+.env,local
+.env.*.local
```

Comment:
These are not for logs (see comment in the top of the file). Please put them at the bottom